server/server_test.go

```diff
@@ -1162,6 +1162,8 @@ func TestConnectErrorReports(t *testing.T) {
 
 	remoteURLs := RoutesFromStr("nats://127.0.0.1:1234")
 
+	s.Shutdown()
```

Comment:
This is deferred from above, why add here? Was test failing?

Reply:
Yes, tests failed. But i have figured it out. We should only `Fatalf` when not in a `shutdown` stage.